Enterprise-CMCS/macpro-mdct-mfp
# Mdct 2815 - State and Territory Specific Initiatives: II. Evaluation Plan

### Description
State and Territory Specific Initiatives: II. Evaluation Plan
This page is showing up in the sidebar right now but will be moved to open up from a dashboard (currently WIP). 
Everything should look like the spec except for the year and quarter numberfield data in the EntityCard -- per the card, we are waiting on BO feedback for that functionality. For the time being that data is not in the EntityCard, and I will create a separate Jira card to address this functionality. Also the hint text on the `yes` radio option has been changed to show up beneath the question itself, and says `Note: If 'Yes', fields use calendar year quarters.`. This is a Design-approved change.


### Related ticket(s)


MDCT-2815

---
### How to test
Create a work plan, go to State and Territory Specific Initiatives: II. Evaluation Plan inside State and Territory Initiatives Instructions, see that you `add initiative` opens a new modal (difference will be in the title), `edit` opens the edit modal, and everything follows the spec. 


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
